packages/rocketchat-message-snippet/server/publications/snippetedMessagesByRoom.js

```diff
@@ -0,0 +1,36 @@
+Meteor.publish('snippetedMessages', function(rid, limit=50) {
+	if (typeof this.userId === 'undefined' || this.userId === null) {
+		return this.ready();
+	}
+
+	let publication = this;
+
+	let user = RocketChat.models.Users.findOneById(this.userId);
+
+	if (typeof user === 'undefined' || user === null) {
+		return this.ready();
+	}
+
+	let cursorHandle = RocketChat.models.Messages.findSnippetedByRoom(
+		rid,
+		{
+			sort: {ts: -1},
+			limit: limit
+		}
+	).observeChanges({
+		added: function(_id, record) {
+			publication.added('rocketchat_message', _id, record);
```

Comment:
We should not use the real collection name here, can you replace by `rocketchat_snippeted_message`